components/favicon_title_handler/favicon_title_handler.tsx

```diff
@@ -83,21 +84,32 @@ class FaviconTitleHandler extends React.PureComponent<Props> {
             currentTeam,
             currentTeammate,
             unreads,
+            isGlobalThreadsView,
         } = this.props;
         const {formatMessage} = this.props.intl;
 
         const currentSiteName = siteName || '';
 
+        const mentionTitle = unreads.mentionCount > 0 ? '(' + unreads.mentionCount + ') ' : '';
+        const unreadTitle = !this.isDynamicFaviconSupported && unreads.messageCount > 0 ? '* ' : '';
+
         if (currentChannel && currentTeam && currentChannel.id) {
             let currentChannelName = currentChannel.display_name;
             if (currentChannel.type === Constants.DM_CHANNEL) {
                 if (currentTeammate != null) {
                     currentChannelName = currentTeammate.display_name;
                 }
             }
-            const mentionTitle = unreads.mentionCount > 0 ? '(' + unreads.mentionCount + ') ' : '';
-            const unreadTitle = !this.isDynamicFaviconSupported && unreads.messageCount > 0 ? '* ' : '';
-            document.title = mentionTitle + unreadTitle + currentChannelName + ' - ' + currentTeam.display_name + ' ' + currentSiteName;
+            document.title = `${mentionTitle + unreadTitle + currentChannelName} - ${currentTeam.display_name} ${currentSiteName}`;
+        } else if (currentTeam && isGlobalThreadsView) {
+            document.title = formatMessage({
+                id: 'globalThreads.title',
+                defaultMessage: '{prefix} Threads - {displayName} {siteName}',
+            }, {
+                prefix: mentionTitle + unreadTitle,
+                displayName: currentTeam.display_name,
+                siteName: currentSiteName,
```

Comment:
Up on :91 okay?

Reply:
@Willyfrog after a second look at this, I think we should leave it as-is. https://github.com/mattermost/mattermost-webapp/blob/master/components/favicon_title_handler/favicon_title_handler.test.jsx#L53-L56 tests specifically for a null sitename. If we do need to change it, I think this should be in a separate PR as it would change non-threading titles.